utils/neuralmagic/utils.py

```diff
@@ -28,6 +29,20 @@
 RANK = int(os.getenv("RANK", -1))
 ALMOST_ONE = 1 - 1e-9  # for incrementing epoch to be applied to recipe
 
+# In previous integrations of NM YOLOv5, we were pickling models as long as they are
+# not quantized. We've now changed to never pickling a model touched by us. This
+# namespace hacking is meant to address backwards compatibility with previously
+# pickled, pruned models.
+import models
+from models import common
+setattr(common, "_Add", _Add)  # Definition of the _Add module has moved
```

Comment:
Hah I love this

Reply:
Was hoping to import it directly in common.py, but quickly found myself in circular import hell. This way is probably more intentional anyway